aten/src/ATen/native/Activation.cpp

```diff
@@ -172,6 +172,19 @@ Tensor & relu_(Tensor & self) {
   return at::threshold_(self, 0, 0);
 }
 
+Tensor swish(const Tensor& self) {
+  return self.mul(self.sigmoid());
+}
+
+Tensor & swish_(Tensor& self) {
+  return self.mul_(self.sigmoid_());
+}
+
+Tensor swish_backward(const Tensor& grad, const Tensor& self) {
+  auto self_sigmoid = self.sigmoid();
+  return grad * (self_sigmoid * (1 + self * (1 - self_sigmoid)));
+}
+
```

Comment:
Please add a .out version, here is an example
https://github.com/pytorch/pytorch/pull/41034/files#diff-8026a1a389bfcf92e484ae272675c336R1187